### PR TITLE
remove deprecated org metric

### PR DESCRIPTION
**Pull request by mstumpfx**
this doesn't exist anymore

### CODE REVIEW

File: elastic-stack/README.md [hunk 1]
```diff
@@ -6,23 +6,23 @@
...
+Create links from Splunk Infrastructure Monitoring charts and alerts to any URL-addressable destination such as Kibana.
 
 The links appear in the Action menus adjacent to metric property values displayed in dashboards and alerts.
 
 The metric property values are used as query parameter and/or path variable values of the URL link.
 
 ### INSTALLATION
 
-Installation is actually configuring linking from property values on SignalFx charts and alerts to any URL-addressable destination. 
+Installation is actually configuring linking from property values on SignalFx charts and alerts to any URL-addressable destination.
```
Comment (jmalin-signalfx) [inline elastic-stack/README.md:17]: To install, add a data link from an Infrastructure Monitoring chart or detector to a URL. To learn more, see the section
[Clicking on a property value to follow a link](https://docs.signalfx.com/en/latest/managing/data-links.html#clicking-on-a-property-value-to-follow-a-link) in the user documentation.

File: elastic-stack/README.md [hunk 2]
```diff
@@ -6,23 +6,23 @@
...
 
 Read about the possible configurations [here](https://docs.signalfx.com/en/latest/dashboards/dashboard-links.html) and priority linking [here](https://docs.signalfx.com/en/latest/managing/data-links.html#local-and-global-data-links).
```
Comment (jmalin-signalfx) [inline elastic-stack/README.md:19]: Unless this feature allows local data linking, or linking to a dashboard, Splunk, or Kibana, this sentence is unnecessary.

File: elastic-stack/README.md [hunk 3]
```diff
@@ -6,23 +6,23 @@
...
 
 To read about creating local data links, click [here](https://docs.signalfx.com/en/latest/managing/data-links.html#local-links).
```
Comment (jmalin-signalfx) [inline elastic-stack/README.md:21]: To learn more about creating local data links, see [Configuring local data links](https://docs.signalfx.com/en/latest/managing/data-links.html#configuring-local-data-links)

Does this feature work with local data links?

**NOTE**: The Splunk Documentation Style Guide specifies that link text for all links should be the same as the heading or sentence that is the link target. In particular, don't use "here" as link text.

File: elastic-stack/README.md [hunk 4]
```diff
@@ -6,23 +6,23 @@
...
 
 You must be an account administrator to configure new global links.
 
-To access Global Data Link configuration options page, click [here](/#/organization/YOUR_SIGNALFX_ORG_ID?selectedKeyValue=sf_section:globaldatalinks). 
+To access Global Data Link configuration options page, click [here](/#/organization/YOUR_SIGNALFX_ORG_ID?selectedKeyValue=sf_section:globaldatalinks).
```
Comment (jmalin-signalfx) [inline elastic-stack/README.md:25]: The link text doesn't conform to the Splunk Documentation Style Guide. See my previous comment.
The link target is not a valid URL.

File: signalfx-org-metrics/README.md.jinja [hunk 5]
```diff
@@ -76,13 +76,12 @@ These metrics track limits that Infrastructure Monitoring enforces for your orga
...
-   Infrastructure Monitoring stops accepting data points for new custom MTS, but it continues to accept
-   data points for custom MTS that already existed. To see the number of MTS you're allowed to have, use the metric
-   `sf.org.numCustomMetrics`. To learn more about custom MTS, see the section
-   [About custom, bundled, and high-resolution metrics](https://docs.signalfx.com/en/latest/admin-guide/usage.html#about-custom-bundled-and-high-resolution-metrics)
-   in the user documentation.
+* `sf.org.limit.customMetricTimeSeries` (gauge): Maximum number of active custom MTS,
+   within a moving window of the previous 60 minutes, that can send data to your organization.
+   If you exceed this limit, Infrastructure Monitoring drops data points for the custom MTS
+   that exceeded the limit, but it continues to accept data points for custom MTS that already existed.
+   you’ve defined, use the metric `sf.org.numCustomMetrics`. To learn more about custom MTS, see the
```
Comment (jmalin-signalfx) [inline signalfx-org-metrics/README.md.jinja:83]: The phrase "you’ve defined, use the metric `sf.org.numCustomMetrics`." is not a sentence.

**Review by jmalin-signalfx — CHANGES_REQUESTED** [review]
See my comments.

**Review by keitwb — APPROVED** [review]

**Pull request merged**